bundles/org.openhab.binding.pixometer/README.md

```diff
@@ -23,7 +23,7 @@ The different meter types are pretty similar in basic, but are implemented in pa
 | --------- | --------------------------------------------- | -------- | ------------- | ------- |
 | user      |                                               | Yes      | -             |         |
 | password  |                                               | Yes      | -             |         |
-| refresh   | Sets the refresh time. Minimum is 60 Minutes. | Yes      | 240           |         |
+| refresh   | Sets the refresh interval. Minimum is 60 minutes. | Yes      | 240           |         |
```

Comment:
It would be nice if this table could be reformatted as it was previously nicely aligned.

Reply:
done